### ISSUE TITLE
Busco issue

### ISSUE THREAD

**Issue by qfaber** [AlexanderLabWHOI/EUKulele#49]
Hi,
I am unable to install the recommended version of Busco via conda and the code is no longer available else so I have a newer version: 5.4.3. I ran EUKulele and I don't think busco worked with the latest version- is there a way I can fix this? Thanks!

**Comment by akrinos** [comment]
Hi @qfaber , we are so excited that you are using `EUKulele`! So sorry for the trouble with `BUSCO`. First question: is it important to you to have the `BUSCO` steps done? If not, I've added a flag to ignore those steps, because of how often that particular piece of the pipeline was problematic. If so, we can talk other potential solutions!

**Comment by qfaber** [comment]
Yes, it is. It seems to be running fine now- I'm getting output folders for taxonomy, but not folders for core_taxonomy or busco_assessment

**Comment by akrinos** [comment]
Hello @qfaber - EUKulele has completed without the BUSCO steps, you mean? If you could send the output messages it got to, that would help with diagnosing. Thanks!

**Comment by qfaber** [comment]
Yes, that's what I mean. I attached the output- it ran for several hours on the final step without doing anything, maybe that was the issue. I can also send error logs if that is helpful. Thanks!
________________________________

**Comment by akrinos** [comment]
The taxonomy estimation steps completed successfully with the dialog you attached?

**Comment by qfaber** [comment]
Yes, for each sample i got files labelled samplename-estimated-taxonomy.out
________________________________

**Comment by akrinos** [comment]
@qfaber sorry for not following up on this. Could you send error logs if you're still having these issues with it? thanks!

**Comment by akrinos** [comment]
Hi @qfaber - let me know if you still need help with EUKulele? Else I'll close up this issue. Thanks!
